### PR TITLE
Update logo doc

### PR DESCRIPTION
**Pull request by nonam4**
## Summary
If name prop is not set logo is not visible, this should be described to user

## Related issue, if any:
#1646 

## What kind of change does this PR introduce?
Docs

## Does this PR introduce a breaking change?
No

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -336,6 +323,8 @@ window.$docsify = {
 
 Website logo as it appears in the sidebar. You can resize it using CSS.
 
+!> Logo will only bee visible if `name` prop is also set. See [name configuration](configuration?id=name).
```
Comment (sy-records) [inline docs/configuration.md:326]: Thanks for the PR.

But need to apply for merge to develop branch, not master. you can resolve the conflict or resubmit a new PR.

**Review by sy-records — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (nonam4) [reply docs/configuration.md:326]: Hello, It's already merging o develop branch

Comment (sy-records) [reply docs/configuration.md:326]: image
It has some conflicts that need to be resolved.

Comment (nonam4) [reply docs/configuration.md:326]: I've created another PR #2347

**Pull request closed without merging**